site/src/theme/light/permission.ts

```diff
@@ -0,0 +1,8 @@
+import type { Permission } from "../permission";
+import colors from "../tailwindColors";
+
+export default {
+	background: colors.zinc[200],
+	outline: colors.zinc[300],
+	text: colors.zinc[700],
+} satisfies Permission;
```

Comment:
I'm wondering if `permission` should be treated as a theme or not as a design token in this case, but I'm okay with it for now. Perhaps having a 'muted' pill variant could be better. 🤔 Wdyt @aslilac ?